### DISCUSSION TITLE
make install-headers-only ?

### DISCUSSION THREAD

**Discussion by hongkongkiwi** [finit-project/finit#304] [Q&A] [answered]
Whats the best way to install headers only from finit so I can build the finit plugins? I couldn't see something like `make install-headers-only` and I did not want to make install as this is my development machine and I don't want to change my init.

**Comment by troglobit** [comment, marked as answer]
Hmm, let me see ... we have a `pkginclude_HEADERS` variable set.  I think the following should do the trick:

    $ cd src/
    $ make install-pkgincludeHEADERS DESTDIR=/tmp/foo
    /usr/bin/mkdir -p '/tmp/foo/usr/include/finit'
    /usr/bin/install -c -m 644 cgroup.h cond.h conf.h finit.h helpers.h log.h plugin.h svc.h service.h '/tmp/foo/usr/include/finit'

> `DESTDIR` is optional.

**Comment by hongkongkiwi** [comment]
Great thanks!
